parlai/crowdsourcing/tasks/turn_annotations_static/turn_annotations_blueprint.py

```diff
@@ -78,12 +78,16 @@ class TurnAnnotationsStaticBlueprintArgs(StaticReactBlueprintArgs):
             "help": "Path to data and answers for onboarding task in JSON format"
         },
     )
-    annotation_buckets: str = field(
-        default=os.path.join(get_task_path(), 'task_config/annotation_buckets.json'),
+    annotation_buckets: Optional[str] = field(
+        default=None,
         metadata={
-            "help": "As per Turn Annotations task, path to annotation buckets which will be checkboxes in the frontend for worker to annotate an utterance."
+            "help": "As per Turn Annotations task, path to annotation buckets which will be checkboxes in the frontend for worker to annotate an utterance. If none provided, no checkboxes."
         },
     )
+    response_field: bool = field(
+        default=False,
+        metadata={"help": "If we want the crowdworker to respond to the message."},
```

Comment:
nit: it might be good to be more specific by saying that this will add a text box, for use by the crowdsource worker in responding to the message